packages/app/src/containers/ColorSwatchesInspector.tsx

```diff
@@ -46,35 +46,41 @@ const ColorPickerInspector = memo(function ColorPickerInspector() {
   const name =
     selectedSwatches.length > 1 &&
     (firstSwatch.name === 'New Color Variable' || !sameName())
-      ? 'Multiple'
+      ? undefined
```

Comment:
If all the selected colors are New I want to show the `Multiple` placeholder anyway. If not, when they choose multiple new colors it will show "New Color Variable" in the textbox. Maybe it's okey if it looks like this